changelog_unreleased/html/pr-8614.md

```diff
@@ -0,0 +1,17 @@
+#### Remove extra linebreaks ([#8614](https://github.com/prettier/prettier/pull/8614) by [@fisker](https://github.com/fisker))
+
+<!-- prettier-ignore -->
+```html
+<!-- Input -->
+<div>
+  Hello, world!
+</div>
+
+<!-- Prettier stable -->
+<div>
+  Hello, world!
+</div>
+
+<!-- Prettier master -->
+<div>Hello, world!</div>
+```
```

Comment:
Maybe we can improve it, it’s not clear what really changed and why?

Reply:
Yes, this needs an explanation. I can try to write it when I have time this weekend.

Reply:
Thank you.